CHANGELOG.unreleased.md

```diff
@@ -6,6 +6,7 @@
 ### Enhancements
 
 > Users must be able to say: “Nice enhancement, I'm eager to test it”
+> [backup] go back to precedent page instead of going to the overview when saving a backup job : keeps the user current filters ang page
 
 ### Bug fixes
 
```

Comment:
Change `xo-web patch` to `xo-web minor` in the "Packages to release" section since this is an actual feature and not only a bug fix.